packages/view3d-optimizer/index.js

```diff
@@ -33,93 +32,94 @@ program.parse(process.argv);
-  const cwd = process.cwd();
-  const tmpFile = `${tempDir.path}/${modelName}.gltf`;
-  const outDir = options.output
-    ? path.resolve(cwd, options.output)
-    : path.resolve(path.dirname(options.input), modelName);
-
-  const run = async (cmd, taskDesc) => {
-    if (!options.silent) {
-      process.stdout.write(`- ${chalk.yellow(taskDesc)}...`);
-    }
-
-    const startTime = performance.now();
-    const tasks = (Array.isArray(cmd) ? cmd : [cmd]).map(task => exec(task));
-    await Promise.all(tasks).catch(err => {
-      process.stdout.write("\n");
-      err.stdout && console.error(chalk.red(err.stdout));
-      err.stderr && console.error(chalk.red(err.stderr));
-      process.exit(1);
-    });
-    const endTime = performance.now();
+const modelName = path.basename(options.input, path.extname(options.input));
+const cwd = process.cwd();
+const tmpFile = path.resolve(cwd, "temp.glb");
+const outDir = options.output
+  ? path.resolve(cwd, options.output)
+  : path.resolve(path.dirname(options.input), modelName);
+const outName = `${modelName}.gltf`;
 
-    if (!options.silent) {
-      process.stdout.clearLine(0);
-      process.stdout.cursorTo(0);
-      process.stdout.write(`- ${taskDesc}... ${chalk.green("DONE!")} (${chalk.blue(`${((endTime - startTime) / 1000).toFixed(2)}s`)})\n`)
-    }
+const run = async (cmd, taskDesc) => {
+  if (!options.silent) {
+    process.stdout.write(`- ${chalk.yellow(taskDesc)}...`);
   }
 
-  await exec(`npx gltf-pipeline -i ${options.input} -o ${tmpFile} -s`);
+  const startTime = performance.now();
+  const tasks = (Array.isArray(cmd) ? cmd : [cmd]).map(task => exec(task));
+  await Promise.all(tasks).catch(err => {
+    process.stdout.write("\n");
+    err.stdout && console.error(chalk.red(err.stdout));
+    err.stderr && console.error(chalk.red(err.stderr));
+    process.exit(1);
+  });
+  const endTime = performance.now();
 
-  if (options.forceJpg) {
-    const gltf = fs.readJSONSync(tmpFile);
-    const images = [...gltf.images].filter(image => image.uri);
-    const imageURIs = images.map(img => path.resolve(tempDir.path, img.uri));
-
-    await run(`npx squoosh-cli --mozjpeg auto --output-dir "${tempDir.path}" ${imageURIs.join(" ")}`, "Converting images to jpg");
-    images.forEach(image => {
-      image.mimeType = "image/jpeg";
-      image.uri = `${path.basename(img.uri, path.extname(img.uri))}.jpg`;
-    });
-
-    // Save modified gltf
-    fs.writeJSONSync(tmpFile, gltf);
+  if (!options.silent) {
+    process.stdout.clearLine(0);
+    process.stdout.cursorTo(0);
+    process.stdout.write(`- ${taskDesc}... ${chalk.green("DONE!")} (${chalk.blue(`${((endTime - startTime) / 1000).toFixed(2)}s`)})\n`)
   }
+}
 
-  if (options.basisu) {
-    await run(`npx gltf-transform etc1s ${tmpFile} ${tmpFile}`, "Applying BasisU (ETC1S) texture compression");
-  }
+const main = async () => {
+  await exec(`npx gltfpack -i ${options.input} -o ${tmpFile}`);
```

Comment:
When installing a module with `npx` for the first time, it goes through a confirmation process in the terminal, but if you use the `npx --yes` option, it will be installed as a slient and usability will be improved.

Reply:
Modules will be installed as a dependency when installing view3d-optimizer, I think it doesn't install again after calling `npx`